### DISCUSSION TITLE
Fully functional example application with FastAPI (Dockerized)

### DISCUSSION THREAD

**Discussion by j0hn-mc-clane** [celery/celery#7045] [Show and tell]
Hi guys

I finally setup an entire example application using FastApi, Poetry pacman, Celery and Flower.
All of this is composed and built using Docker.

I struggled a lot to find a summarized blog to implement it.

I decided to share with you the example application: https://github.com/JCardoen/fast-api-example

Feel free to create PRs, fork it etc etc

General feedback is defo welcome!

Take care.

Fellow dev

**Comment by auvipy** [comment]
thank fellow dev
